src/event/mod.rs

```diff
@@ -26,8 +26,10 @@ pub mod proto {
 
 pub static LOG_SCHEMA: OnceCell<LogSchema> = OnceCell::new();
 
+pub const PARTIAL_STR: &str = "_partial"; // TODO: clean up the _STR suffix after we get rid of atoms
```

Comment:
We've gotten rid of Atoms at least on some parts of the code. Can this be removed now? (Noting the TODO)

Reply:
Nope, this is fresh. Need it cause `::get` takes atoms still.